HOW_TO_RELEASE.md

```diff
@@ -0,0 +1,62 @@
+# Preparing a pyproj release
+
+Preparing a pyproj release is a two-phase process.
+
+## Phase 1: Release Candidate
+
+In this phase, we want to ensure all the builds work on all platforms and methods
+of distribution for the next release.
+
+### Add the rc postfix
+
+The first step in this phase is to update the version number `__version__` in `__init__.py`
+to the next release `<major>.<minor>.<patch>`. Then, add the `rc` style posfix following the [PEP-440](https://www.python.org/dev/peps/pep-0440/#pre-releases) conventions.
+
+### Create a tag on the repository
+
+The next step is to create a tag with the same name as the version just added. This can be done using the git command line or from https://github.com/pyproj4/pyproj/tags.
```

Comment:
small nit: it's not exactly the same, but prefixed with "v"

Reply:
Good catch. Thanks!